crm_stage_multi_team/models/crm_lead.py

```diff
@@ -45,14 +45,14 @@ def _read_group_stage_ids(self, stages, domain, order):
         )
         if test_condition or self.env.context.get("no_crm_stage_multi_team"):
             return super()._read_group_stage_ids(stages, domain, order)
-        team_id = self._context.get("default_team_id")
-        if team_id:
+        team_ids = self.env["crm.team"]._get_default_team_id().ids
```

Comment:
Isn't this returning only one team?

Reply:
```suggestion
        team_ids = self.env.user.crm_team_ids.ids
```
You are right. Could I get the teams direct from the user or should i override the method `_get_default_team_id`?

Reply:
Don't change `_get_default_team_id`. This is something exclusive to this specific method.